examples/hacker_news_pwa/lib/src/feed_component.html

```diff
@@ -1,6 +1,6 @@
 <!-- Loading view -->
 <template [ngIf]="items == null">
-  <div>Loading...</div>
+  <div class="loading">Loading...</div>
```

Comment:
If you're going to try and style up these loading state let's do the same for zero state. Check out the Jobs feed and click next until you get an empty page. Perhaps give this class a less-loading specific name and apply it to the div below and share the styles?